envCheck.sh

```diff
@@ -1,6 +1,7 @@
 # This script is used to assert if require env vars are present for deployment
 # RPC_URL: RPC URL for the network
 # ACCOUNT: Accessed with `cast wallet` command
+# ETH_FROM: Address to send transactions from TODO: remove this after bug in foundry is fixed
```

Comment:
Can you add a link to the bug here?
It's unclear what this is about

Reply:
addressed in 497fb15